app/templates/_package.json

```diff
@@ -3,6 +3,22 @@
   "engines": {
     "node": ">=4"
   },
+  "dependencies": {
+    <% if (includeModernizr) { %>
+    "modernizr": "^3.6.0",
```

Comment:
This line will still have a comma `,` when I scaffold without jQuery and bootstrap. 
Will throw `npm ERR! JSON.parse Failed to parse json`.

Reply:
Let me fix that! 👍